docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
 | Upsample | No |
 | Where | No |
 | Xor | No |
+
+
+## PaddlePaddle\* Supported Operators
+
+Standard PaddlePaddle(paddlepaddle>=2.1)\* Operators:
+
+| Operator Name in PaddlePaddle\*| Limitations|
+| :----------| :----------|
+| adpative_pool2d | Only supports the NCHW data_layout |
+| arg_max | Only supports the 'Int64' output data_type |
+| assign_value | No |
+| batch_norm | No |
+| bilinear_interp | Only supports the NCHW data_layout |
+| bilinear_interp_v2 | Only supports the NCHW data_layout |
+| bmm | No |
+| cast | No |
+| clip | No |
+| concat | No |
+| conv2d | Only supports the NCHW data_layout |
+| depthwise_conv2d | Only supports the NCHW data_layout |
+| deformable_conv | No |
+| elementwise_add | No |
+| elementwise_div | No |
+| elementwise_max | No |
+| elementwise_min | No |
+| elementwise_mul | No |
+| elementwise_pow | No |
+| elementwise_sub | No |
+| equal | No |
+| expand_v2 | No |
+| fill_constant_batch_size_like | Only supports the 'FP32' data_type |
```

Comment:
Should remove the limitation

Reply:
done, updated limitations of 'fill_constant_batch_size_like', 'slice', 'unsqueeze2'